### ISSUE TITLE
Form Fundraising WG

### ISSUE THREAD

**Issue by jacobian** [django/dsf-working-groups#9]
The [Fundaising WG](https://github.com/django/dsf-working-groups/blob/main/active/fundraising.md) has been formally approved by the Board but right now it's just a placeholder. Broadly speaking we know we want to form a much larger group to work on fundraising strategy, but beyond that things are pretty open.

I'm opening this issue to track two things:

- interest: if folks are interested in serving on this WG, please express interest here
- Progress on getting the group up and running

Right now (Jan 10 2024), we're essentially waiting on someone(s) to pick up and flesh out what the group does, who will be on it, etc. (See what @sarahboyce did with the [Social Media WG](https://github.com/django/dsf-working-groups/pull/6) as a template for what needs to happen.)

I'd intended on taking lead here, but likely won't get around to it for ... a few months, if not longer. So if someone else is motivated to get things moving, that'd be great, dive in! I'd be very happy to guide/mentor someone if you're interested and have time but aren't sure where to start.

**Comment by priyapahwa19** [comment]
Hey @jacobian,
This looks interesting. I would love to work on this :)

**Comment by sabderemane** [comment]
Hi there, I open a draft PR #14 to start something, I can't tag everyone as reviewers, FYI @priyapahwa @StephanieAG @knyghty when you have some time :)

**Comment by StephanieAG** [comment]
Hi all 👋 I could commit to at least 2-4 hours/month to help the FWG, but I'm struggling how to help right now. 

I know the next step is finalizing the draft PR #14 Sarah posted (thanks, Sarah!), but I'm struggling to understand how to use GitHub with other people. For context, I've only been learning Python/Django for around a year and my day job is completely unrelated to coding so I've never worked with other people on something like this, and I'm not super familiar with the tools for collaboration yet. I poked around and I will search the internet and learn more, but in the meantime I don't want to mess anything up. 

Anyone want to join me for a "review PR 14 co-working session" where we can review this together and someone can show me how not to make a fool of myself while collaborating on GitHub 😜? 

If this appeals to anyone: what timezone are you in and when are you available? My schedule is flexible, and I'm on the East coast of the USA, so my timezone is UTC-04:00. I can set up the videoconference link. I'm thinking either 30 minutes or an hour.

**Comment by jacobian** [comment]
🖐🏻 I'd love to come and help as much as I can! I still haven't had a chance to review the PR in depth, but I can commit to doing so ahead of a co-working session if we get one on the books. I'm in the US-Pacific time zone (UTC-7), but don't mind early mornings to meet up with East Coast times. Generally speaking Thursday-Sundays are best -- M/Tu/W I'll usually have work conflicts during normal work hours. Not _this_ weekend, unfortunately, so maybe towards the end of next week?

**Comment by StephanieAG** [comment]
Lovely, thank you 😁 
Proposing a few options below. I'll plan for 60 minutes, but we can always stop early.

1. Fri 3/22 — **9am PT - UTC-7** (12pm ET - UTC-4)
2. Sun 3/24 — **10am PT - UTC-7** (1pm ET - UTC-4)
3. Thu 3/28 — **3pm PT - UTC-7** (6pm ET - UTC-4)
4. Fri 3/29 — **11am PT - UTC-7** (2pm ET - UTC-4)
5. Another time on Thu/Fri/Sun — I'm also flexible on these days, so feel free to propose a different time

All welcome; feel free to let us know what option/s work for you, and I can post a videoconference link after a time is determined.

**Comment by jacobian** [comment]
1/2/3 all work for me - just not the 29th. Pick whichever one you want and let's make it happen!

**Comment by StephanieAG** [comment]
Great, I went with 3. Link to join below; see you then!

Review PR #14 Coworking Session - Django Fundraising Working Group
Thu 3/28 — 3pm PT - UTC-7 (6pm ET - UTC-4)
Video call link: [https://meet.google.com/kgy-zray-map]

**Comment by priyapahwa** [comment]
Hey @StephanieAG 
Is it possible to go either by 1 or 2? I was hoping to join then but 3rd one is 3:30 am as per IST :|

**Comment by StephanieAG** [comment]
Sure but even option 1 still seems pretty late for you (9:30pm IST), and @jacobian previously mentioned not minding early mornings, so I took the liberty of updating the time to **Fri 3/22 at 8am PT / 11am ET / 8:30pm IST** (one hour earlier than option 1). Obviously just let me know if that's not preferable for either of you.

Review PR #14 Coworking Session - Django Fundraising Working Group
Fri 3/22 at 8am PT / 11am ET / 8:30pm IST
Video call link: https://meet.google.com/kgy-zray-map

**Comment by priyapahwa** [comment]
Works for me, thanks :)

**Comment by jacobian** [comment]
WFM too see you then

**Comment by StephanieAG** [comment]
Hey friends, I know we're waiting on some reviews on the pull request and then the board approval to "officially" start but I was thinking I could maybe still start doing something now to help us all hit the ground running when we do get this working group going. 

I don't want to overstep and I don't have as much historical / behind-the-scenes knowledge of anything that's been already done in this area, so I wanted to propose a handful of ideas that I could start working on:

- setting up some **informational interviews** of people in the community who have been around a while to get a sense of the history (what's been done, what's been tried, what is the present process, what thoughts have been long-standing but haven't been tried yet, etc.) and then compiling notes from those interviews and summarizing
- given the long-term thought of potentially hiring a DSF executive director or other paid fundraising help (and even if we don't), **putting together a CRM** would be helpful/necessary. I'm still not super clear on what the current state of anything is, but, if it's needed, I can start gathering the information we do have into one place and organizing it. For this, I'd need to talk to someone with access to the various locations of existing info (I'm thinking probably I'd want to start with Catherine Holmes who I gather knows a ton, but, again, I'm newer so recommendations welcome).
- **grant opportunity hunting / fact-finding** (I would probably start by looking more at the three prospects in [our gsheet](https://docs.google.com/spreadsheets/d/14gEzRuumXDEMDIc7tabM-fZfIdBw4TrnNiibw1D38cg/edit#gid=0) for important info like deadlines/requirements/etc. and then add that info. to our tracker)
- something else? 

In short, I'm asking: what would be most helpful for me to spend time on right now? I'd be more than happy to start working on any of the above.

**Comment by StephanieAG** [comment]
I'm going to just go for working on the third idea on this list (about looking at grant opportunities) because it seems unlikely anyone would have a problem with me starting that. 

Feel free to let me know if you feel I should start with something else.
